store/project.go

```diff
@@ -452,7 +463,7 @@ func patchProjectImpl(ctx context.Context, tx *sql.Tx, patch *api.ProjectPatch)
 		UPDATE project
 		SET `+strings.Join(set, ", ")+`
 		WHERE id = $%d
-		RETURNING id, row_status, creator_id, created_ts, updater_id, updated_ts, name, key, workflow_type, visibility, tenant_mode, db_name_template, role_provider
+		RETURNING id, row_status, creator_id, created_ts, updater_id, updated_ts, name, key, workflow_type, visibility, tenant_mode, db_name_template, role_provider, schema_migration_type
```

Comment:
We would need to do if-else branching for dev/release mode. This can come from "s.db.mode".

https://github.com/bytebase/bytebase/blob/main/docs/schema-update-guide.md#development

Reply:
fixed.